docs/source/sandbox/ReproNim Entities.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.59 Safari/537.36" version="5.7.1-3" editor="www.draw.io" type="device"><diagram>7Vxbc5s4FP41fnSGu+3H5tJ2Z9KZTLOzu33qKCDbagAxQk7i/voVIIGEwME2sWlCHlo4PpKQzu07B4mJfRW9fCEgWX/DAQwnlhG8TOzriWU5lmGw/zLKtqCYc88tKCuCAk6rCPfoN+RE3nC1QQFMFUaKcUhRohJ9HMfQpwoNEIKfVbYlDtVRE7CCGuHeB6FO/RcFdF1Q565R0b9CtFqLkU0x4wfgP64I3sR8vIllL/O/4ucIiL44f7oGAX6WSPbNxL4iGNPiKnq5gmG2uGLZinafW34tn5vAmHZp4M2LFk8g3PC5X+F4iVYbAijC8V8URvxJ6VasTvqMohDE7O5yTaOQEU126a9RGNyCLd5kI6eUrYS4u1xjgn7jmALBzH4mlMvdNhSO+6wlI2fUJQrDKxxikg9sG8bcZ0vJ6IxTon/O/xg9X3kY8EEyLj6GORP3fBpG/hAEP5YCtrL2MGX8d2L5jJJ0C1IqHl2ILPs1AOk6Hy+7ASFaxezaZ60he7RLXRpcQE+QUPgikbh0vkAcQUq2jIX/OnW4pnBTmvPb50otLcGyllRyLjQMcFNYlV1X6sAuuEa0aMdC046J5YEoYXOLH9KknKKkHM9rROF9Avzs/pn5B1VLxBqFcElLEQhRxjhXKlnqnJSyDlG8us1bXTsNws2WFDEL/sQHoDgR0gMPpcgJpoBK90w2Qq66sErjeF1aXDp2R+nMFj1IZ2Friw8D5sb4LSZ0jVc4BuFNRZUsxFAFk1vkp8x3Sque0T6j7AlydhgHguPmewTibUGTOH5BSrdcLGBDMSNVz3GLM5nko7E1Jtv/skYXtuMJwo+csFgsBOEOEsTWJTOm4gFeEC1azVx+++NV8aV4Q3yoajSb2ApSlZYt3k4hExgyp/ikxogmCeZN2UqBrcSQYBTTVOr5LiNUlm7OVUtf1F13jX9u72JnF8UDVKpUzqSbdjma7V/DhEkbxv5WUzxmH1TVp7rl8ekzbvdy4l43ucu6CUcoCHKlDcEDDC/L0Cp5fh5cm8QvzEOz3hI28KdTQm+jDzYuDNtxleXmPe2nEpoMp5ba6ULtAC+XKdPSuk/YT5CWJsc7tpIZABrj+vniumq8pu1qgcP0GgJHyXhM4LA1jWBRPczkuWEXq+yCsXIS66ykClqAngQpBhGUWKVfGpjZ8qQMVHbmFyQW/WNBA8TPEIZPN0QeWGbZu/8H8lrT9wtyrM6a245xmlXV6EFVZw2qugPzsFkhuv2eO2Ucy7hHko6CYn5Dgv/G3+pYxpTAhnlhGJ6ENowL13P4vQZQKlzDG1W4xnRLQr1Zq0ORsYutQ5euyEWWldEoK6OXeGaqEMY0azC3mA5vtAPbOIbakVvrp1gDrZ8DoqOppzhJER5/rlkM+BOAjrCRo3EO09mZaMoXfuocpxe9Ihm9VvG1SUQjjDkhjLFVQ7W6Boc+yhON1YkDcQzdJnvgmHzEDwcV+iiHNGpDH9UQswnV9ggV+MKr5Y4Dw31VxlgsBJIoGnksa21GFkp9poZb5DpNs+i6wQnT7YgnuPTr8d4yHF3gO/FGd2ihjTWrRaHO2EKE/DLpfjNwIZxcA7gIEDN19LCheT40fJBhHl9NORJbqvUXS23eT6XEe1sXItVMudvIKkuWW/MbjnNUmiCMWDZsYeyDSRSmbk+ZQr2ft7NlPRWVbfgngQlOEcXkjyiOlqreR9LgmLWk4cjiaL9Grdc/v7eLaswdTpc7ePXcwTtl8nC+KuiGhB8vdyjNcJDJg31ooG99nVoWDxV4X8X6lqTByNe/MfjXEoBIh/61V7SNGUvXF7TdoIbTADW65hCdo0PnDLDhjeUulD36+vPViczZKUsD7rlcffcXTOdp+n7jTekMeo83Xh/gQ3dV/KX8NxCzf8norAYETMvk9BTOyjqVs3q/tm8N2vZt3fYZPht34wxnN47dFZz0kojafRp8gc5/Jtk+865wIYApRXGe2uzXcMQZkjUPM69tekXaa0V7x/6ZVrtU3kI1VKtt/qJxONXqudOCCPYtV09n9Z7erGAtBlKOdkQRiIMx1Awm1LgnrXk6pqYSh4caXyjTR3P5pWEN0uU7usu/iZ8QwXHEZvAPICif5+gChuICZp031PbhAlw9KhzuAh7hdtwz9YqzWHTWjjM4C0/PRW9ekvzFA8cfo484j4+YerX9TqdNSr0jktL3a8zeoJM9T9++JEX+0ZrPac2l9Z7HmpuO0YzW3LYHahDWLLa39rclYe8NBEee8W61G+VANldN5UD22bYQ2LqhfMrOGz6NCdOQdg+cuECvnzs6PGXasfmr6UxK0wnYfY+9vl8PbrcdfRuEB1+4Oz34oL/PcbjvHsbHNKZm7byAw7/f0FqYrzdwDbcm7L2+p8Fuq684FezVt7Lsm/8B</diagram></mxfile>
```

Comment:
I wondered if there is a way to save it in more git friendly way? Eg formatted into multiple lines etc...?